common/test/acceptance/tests/helpers.py

```diff
@@ -215,6 +216,22 @@ def select_option_by_value(browser_query, value):
     select = Select(browser_query.first.results[0])
     select.select_by_value(value)
 
+    def options_selected():
+        """
+        Returns True if all options in select element where value attribute
+        matches `value`. if any option is not selected then returns False
+        and select it.
+        """
+        all_options_selected = True
```

Comment:
@jzoldak handling the scenario now when given value has no corresponding option.